### PR TITLE
crds: do not log if crds are up-to-date

### PR DESCRIPTION
**Pull request by acardace**
Signed-off-by: Antonio Cardace <acardace@redhat.com>

**What this PR does / why we need it**:

This message gets logged way too many times and it pollutes the logs with no useful info.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/virt-operator/resource/apply/crds.go [hunk 1]
```diff
@@ -113,8 +113,6 @@ func (r *Reconciler) createOrUpdateCrd(crd *extv1.CustomResourceDefinition) erro
 			return err
 		}
 		log.Log.V(2).Infof("crd %v updated", crd.GetName())
-	} else {
-		log.Log.V(2).Infof("crd %v is up-to-date", crd.GetName())
```
Comment (ashleyschuett) [inline pkg/virt-operator/resource/apply/crds.go:117]: lets instead change this to log level 4 to match other resources

Comment (acardace) [reply pkg/virt-operator/resource/apply/crds.go:117]: updated, thanks!

**Review by ashleyschuett — APPROVED** [review]

**Pull request merged**